GTBitsOfGood/focus-ga
# email alerts

### Description

Closes (https://github.com/GTBitsOfGood/focus-ga/issues/43)



### Type of change


🤕 Bug Fix
🆕 Feature
🔁 Chore/Cleanup
✨ Refactor

### How to Test

1. comment to add notification objects
2. verify that notification objects are not created for users with no notification preference 
3. netlify dev
4. netlify functions:invoke notif to send email

### Checklist
